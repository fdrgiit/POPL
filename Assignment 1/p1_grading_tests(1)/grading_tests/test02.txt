WHILEwhile whileWHILE WHILE 



while IFWHILE if IF

A02.0INT

00001

<<<<<
>>
