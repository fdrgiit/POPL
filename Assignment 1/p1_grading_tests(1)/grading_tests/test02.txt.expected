1 37 WHILEwhile 
1 37 whileWHILE 
1 6 WHILE
5 37 while 
5 37 IFWHILE 
5 37 if 
5 5 IF
7 35 A02.0INT 
9 34 0 
9 34 0 
9 34 0 
9 34 0 
9 34 1 
11 31 <<
11 31 <<
11 28 <
12 32 >>
